### PR TITLE
add legend.disabled

### PR DESCRIPTION
**Pull request by cuijian-dexter**
```
                    legend: {
                        orient: 'vertical',
                        left: 'left',
                        disabled:{
                            '邮件营销':true
                        },
                        data: ['直接访问','邮件营销','联盟广告','视频广告','搜索引擎']
                    },
```

### CODE REVIEW

File: src/component/legend/LegendModel.js [hunk 1]
```diff
@@ -169,10 +170,14 @@ var LegendModel = echarts.extendComponentModel({
      */
     toggleSelected: function (name) {
         var selected = this.option.selected;
+        var disabled = this.option.disabled;
         // Default is true
         if (!selected.hasOwnProperty(name)) {
             selected[name] = true;
         }
+        if (!disabled.hasOwnProperty(name) && disabled[name] === true){
```
Comment (earlymeme) [inline src/component/legend/LegendModel.js]:
```suggestion
        if (!disabled.hasOwnProperty(name) && disabled[name] === true) {
```

**Comment by Ovilia** [comment]
@100pah Please have a review.

Also, should those disabled legends have a default difference in appearance? Like opacity?

**Comment by Ovilia** [comment]
@cuijian-dexter 我用这个例子测的，怎么好像没效果？（感觉这样的事已经不是第一次了……

```
option = {
                "yAxis": [{
                    "type": "value"
                }],
                "xAxis": [{
                    "data": ["Amaretto", "Caffe Mocha", "Columbian", "Darjeeling", "Decaf Irish Cream", "Green Tea"],
                    "type": "category"
                }],
                "legend": {
                    "data": ["cb"],
                    disabled: {
                        cb: true
                    }
                },
                "series": [{
                    "data": [{
                        "name": "Amaretto",
                        "itemStyle": {},
                        "value": 89
                    }, {
                        "name": "Caffe Mocha",
                        "itemStyle": {},
                        "value": 191
                    }, {
                        "name": "Columbian",
                        "itemStyle": {},
                        "value": 83
                    }, {
                        "name": "Darjeeling",
                        "itemStyle": {},
                        "value": 45
                    }, {
                        "name": "Decaf Irish Cream",
                        "itemStyle": {},
                        "value": 95
                    }, {
                        "name": "Green Tea",
                        "itemStyle": {},
                        "value": 102
                    }],
                    "name": "cb",
                    "type": "bar"
                }],
                "color": ["#c23531", "#2f4554", "#61a0a8", "#d48265", "#91c7ae", "#749f83", "#ca8622", "#bda29a", "#6e7074", "#546570", "#c4ccd3"]
            };
```

**Comment by cuijian-dexter** [comment]
@Ovilia 重新修复了一下，是判断有点问题

**Pull request closed without merging**